README.md

```diff
@@ -1,27 +1,30 @@
-# Alpha Status
+# F5 Networks Helm Charts
 
-This project is presently in alpha status and chart availabiliy in the helm repos below is limited and may be unstable.
+This repository contains [helm](https://docs.helm.sh/using_helm/#using) charts for use with some [F5 Networks](https://f5.com/) products and services within a [Kubernetes](https://kubernetes.io/) or [OpenShift](https://www.openshift.com/) environment.
 
-# F5 Networks Helm Charts
+**Note** that charts may require access to `kube-system` namespace and/or cluster wide permissions to function as expected, so be sure to install/configure helm/tiller appropriately.
+
+## Stable Charts
 
-To use the charts in this repository to deploy the [F5 k8s-bigip-ctlr](https://github.com/F5Networks/k8s-bigip-ctlr) using [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) or [ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/):
+To add the stable repo to helm:
 
 ```
 helm repo add f5-stable https://f5networks.github.io/charts/stable
-helm install f5-stable/f5-bigip-ctlr
 ```
 
-To access additional charts in beta:
+These charts are tested and documented:
+- [f5-bigip-ctlr](https://github.com/F5Networks/charts/tree/master/src/stable/f5-bigip-ctlr) - Use this chart to deploy the [k8s-bigip-ctlr](http://clouddocs.f5.com/products/connectors/k8s-bigip-ctlr/latest/) in Kubernetes or OpenShift.
+- [f5-bigip-ingress](https://github.com/F5Networks/charts/tree/master/src/stable/f5-bigip-ingress) - Use this chart to deploy ingress resources in Kubernetes or OpenShift.
 
-```
-helm repo add f5-incubator https://f5networks.github.io/charts/incubator
-```
+## Documentation
 
-## Repository Structure
+Each chart contains its own `README.md` and the `values.yaml` file includes notes on default values and links to the core documentation for the resources each chart is deploying.
 
-This repo contains packaged and versioned charts and their source. 
+## Incubation Charts
 
-Charts in `/stable` have been tested and documented, charts in `/incubator` may not yet be fully tested and documented. 
+To access additional charts in a development or testing mode that may not be documented:
```

Comment:
Again, taken from the f5 heat repos, feel free to amend/ignore:
```
The incubation charts may have been created by F5 Networks or by external contributors. These charts have not undergone full testing and are subject to change. F5 Networks does not provide technical support for templates in the incubation directory.

To access charts in the incubation area: 

```

Reply:
Like it. Used it. Done :)